packages/playwright-core/src/utils/registry.ts

```diff
@@ -738,7 +738,7 @@ export function buildPlaywrightCLICommand(sdkLanguage: string, parameters: strin
     case 'csharp':
       return `pwsh bin\\Debug\\netX\\playwright.ps1 ${parameters}`;
     default:
-      return `npx playwright ${parameters}`;
+      return `npx playwright@${getPlaywrightVersion()} ${parameters}`;
```

Comment:
Don't do this. If we detect running `npx playwright install` from non-local, print a nice error message (but do not crash), that warns the user and includes a hit at the more future-proof way.